src/components/navMenu/mobileNavMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import {
+  Text,
+  Link,
+  Icon,
+  Button,
+  useCloseDropDown,
+  List,
+} from "@nypl/design-system-react-components";
+import React, { useRef, useState } from "react";
+import FocusLock from "@chakra-ui/focus-lock";
+import { Box, chakra } from "@chakra-ui/react";
```

Comment:
You can import `Box` from the DS. Why did you need to wrap the `MobileNavMenu` component in the `chakra` factory function?

Reply:
Holdover from first attempt at focus lock